tests/playwright/src/ai-lab-extension.spec.ts

```diff
@@ -19,28 +19,31 @@
 import type { Locator, Page } from '@playwright/test';
 import { expect as playExpect } from '@playwright/test';
 import { afterAll, beforeAll, beforeEach, describe, test } from 'vitest';
-import type { DashboardPage, SettingsBar, RunnerTestContext } from '@podman-desktop/tests-playwright';
+import type { DashboardPage, ExtensionsPage, RunnerTestContext } from '@podman-desktop/tests-playwright';
 import {
   NavigationBar,
-  SettingsExtensionsPage,
   WelcomePage,
   PodmanDesktopRunner,
+  removeFolderIfExists,
 } from '@podman-desktop/tests-playwright';
 
 const AI_LAB_EXTENSION_OCI_IMAGE: string = 'ghcr.io/containers/podman-desktop-extension-ai-lab:nightly';
+const AI_LAB_CATALOG_EXTENSION_LABEL: string = 'redhat.ai-lab';
+const AI_LAB_NAVBAR_EXTENSION_LABEL: string = 'AI Lab';
+const AI_LAB_PAGE_BODY_LABEL: string = 'Webview AI Lab';
 
 let pdRunner: PodmanDesktopRunner;
 let page: Page;
 
 let navigationBar: NavigationBar;
 let dashboardPage: DashboardPage;
-let settingsBar: SettingsBar;
-let extensionsPage: SettingsExtensionsPage;
+let extensionsPage: ExtensionsPage;
 
 beforeAll(async () => {
-  pdRunner = new PodmanDesktopRunner();
+  await removeFolderIfExists('tests/output/images');
```

Comment:
Do you really need to delete this folder? In bootc it's used to store the created bootc images, but I dont think you are using it, so you dont really need to initially delete it.

Reply:
Addressed, retested.